tests/integration/targets/setup_rabbitmq/tasks/ubuntu-1.yml

```diff
@@ -12,11 +12,14 @@
         Pin: version 1:20.3.6
         Pin-Priority: 1000
 
-- name: Install https transport for apt
+- name: Install https transport for apt and sudo
   apt:
-    name: apt-transport-https
+    name: "{{ item }}"
```

Comment:
A point of style, but I won't block this PR on it.

This is a list item, so this would execute faster if it was expressed as:

  apt:
    name:
      - apt-transport-https
      - sudo

Reply:
@aitorpazos would you like to change the tests how @odyssey4me 's suggesting?

Reply:
Yeah, you are right... pushed